script.xbmc.lcdproc/lcdmain.py

```diff
@@ -0,0 +1,144 @@
+'''
+    XBMC LCDproc addon
+    Copyright (C) 2012-2018 Team Kodi
+    Copyright (C) 2012-2018 Daniel 'herrnst' Scheller
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+'''
+
+import xbmc
+import xbmcaddon
+import time
+import os
+
+__settings__   = xbmcaddon.Addon(id='script.xbmc.lcdproc')
+__cwd__        = __settings__.getAddonInfo('path')
+__icon__       = os.path.join(__cwd__, "resources", "icon.png")
+__scriptname__ = "XBMC LCDproc"
+
+BASE_RESOURCE_PATH = xbmc.translatePath( os.path.join( __cwd__, 'resources', 'lib' ) )
+sys.path.insert(0, BASE_RESOURCE_PATH)
```

Comment:
Why do you need this? Just store an empty  `__init__.py` in resources.lib and you are able to import it elsewhere in your addon (for python2)